examples/compose/plot_column_transformer_mixed_types.py

```diff
@@ -173,40 +179,47 @@
 # hyperparameters as part of the ``Pipeline``.
 # We will search for both the imputer strategy of the numeric preprocessing
 # and the regularization parameter of the logistic regression using
-# :class:`~sklearn.model_selection.GridSearchCV`.
+# :class:`~sklearn.model_selection.RandomizedSearchCV`. This type of
+# hyperparameter search allows to define the time budget through the `n_iter`
+# parameter. Note that you can increase the `n_iter` parameter if you want to
+# test more combinations. Alternatively, one can use the
```

Comment:
I do not think we can say "time budget". Some parameter combinations are slower than others and `RandomSearchCV` does not account for that.

```suggestion
# :class:`~sklearn.model_selection.RandomizedSearchCV`. This
# hyperparameter search randomly selects a fixed number of parameter
# settings configured by `n_iter`. Alternatively, one can use the
```

Reply:
Make sense. A computational budget would have been more appropriate but the suggestion is fine with me,